site/gatsby-site/src/components/discover/Controls.js

```diff
@@ -40,7 +55,19 @@ const Controls = ({ query }) => {
         <Col className="col-auto">
           <DisplayModeSwitch />
         </Col>
-        <FlexGap />
+        <Hbox>
+          <Form.Check
+            type="switch"
+            id="hide-duplicates"
+            checked={hideDuplicates}
+            onClick={(event) => {
+              setHideDuplicates(event.target.checked);
+            }}
+          />
+          <Form.Label for="hide-duplicates">
+            {t('1st report only').split(' ').join('\u00a0')}
```

Comment:
What's the split for? Not sure it will place nice with translations 🤔

Reply:
`\u00a0` is a non-breaking space. The label text sometimes gets split into two lines when it's just regular spaces, which looks bad.

I've been assuming that the t() function just returns the translation as a string, in which case it seems like it would work for any language. Even for those that don't use spaces between words, it will just return the same string. Are there any edge-cases I'm missing?

Reply:
Sorry forgot  about this pr, but you can achieve something similar with one of the CSS rules `break-word` or `word-wrap` or something similar (I always have to google the exact names 😆), and it will be less obscure.